tests/integration/clientv3/experimental/recipes/v3_lock_test.go

```diff
@@ -21,7 +21,7 @@ import (
 	"time"
 
 	"go.etcd.io/etcd/api/v3/mvccpb"
-	"go.etcd.io/etcd/client/v3"
+	clientv3 "go.etcd.io/etcd/client/v3"
```

Comment:
nit: it isn't necessary. The following code can still use `clientv3` without defining a packageName here.

Reply:
It can, but go editor tooling "fixes" it automatically for clarity, can drop it if etcd doesn't like including those tho